components/popover/style/index.less

```diff
@@ -65,7 +65,7 @@
     line-height: 32px;
     height: 32px;
     border-bottom: 1px solid @border-color-split;
-    color: @heading-color;
+    color: @popover-color;
```

Comment:
In the theme we're using, `@heading-color` is white, `@popover-bg` is a light grey and `popover-color` is black.

It doesnt make sense in my opinion to control the text colors of popovers with multiple variables.